x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.tsx

```diff
@@ -139,45 +168,139 @@ export class APIKeysGridPage extends Component<Props, State> {
         <EuiPageContentHeader>
           <EuiPageContentHeaderSection>
             <EuiTitle>
-              <h2>
+              <h1>
                 <FormattedMessage
                   id="xpack.security.management.apiKeys.table.apiKeysTitle"
                   defaultMessage="API Keys"
                 />
-              </h2>
+              </h1>
             </EuiTitle>
-            {description}
+            <EuiText color="subdued" size="s" data-test-subj="apiKeysDescriptionText">
+              <p>
+                {isAdmin ? (
+                  <FormattedMessage
+                    id="xpack.security.management.apiKeys.table.apiKeysAllDescription"
+                    defaultMessage="View and invalidate API keys. An API key sends requests on behalf of a user."
+                  />
+                ) : (
+                  <FormattedMessage
+                    id="xpack.security.management.apiKeys.table.apiKeysOwnDescription"
+                    defaultMessage="View and invalidate your API keys. An API key sends requests on your behalf."
+                  />
+                )}
+              </p>
+            </EuiText>
+          </EuiPageContentHeaderSection>
+          <EuiPageContentHeaderSection>
+            <EuiButton {...reactRouterNavigate(this.props.history, '/create')}>
+              <FormattedMessage
+                id="xpack.security.management.apiKeys.table.createButton"
+                defaultMessage="Create API key"
+              />
+            </EuiButton>
           </EuiPageContentHeaderSection>
         </EuiPageContentHeader>
 
+        {this.state.createdApiKey && !this.state.isLoadingTable && (
+          <>
+            <EuiCallOut
+              color="success"
+              iconType="check"
+              title={i18n.translate('xpack.security.management.apiKeys.createSuccessMessage', {
+                defaultMessage: "Created API key '{name}'",
+                values: { name: this.state.createdApiKey.name },
+              })}
+            >
+              <p>
+                <FormattedMessage
+                  id="xpack.security.management.apiKeys.successDescription"
+                  defaultMessage="Copy this key now. You will not be able to view it again."
+                />
+              </p>
+              <SelectableTokenField
+                options={[
+                  {
+                    key: 'base64',
+                    value: btoa(concatenated),
+                    icon: 'empty',
+                    label: i18n.translate('xpack.security.management.apiKeys.base64Label', {
+                      defaultMessage: 'Base64',
+                    }),
+                    description: i18n.translate(
+                      'xpack.security.management.apiKeys.base64Description',
+                      {
+                        defaultMessage: 'Format used to authenticate with Elasticsearch.',
+                      }
+                    ),
+                  },
```

Comment:
I like the idea but am worried that it could be confusing to have one option called "Elasticsearch" since they're all API keys for Elasticsearch. "Base64" is referenced throughout our docs as a format which is why I used it as a term here. 

Once our products are aligned, I'd like to remove the entire dropdown menu and only show the base64 encoded API key.